app/components/LocaleTime.tsx

```diff
@@ -26,7 +26,7 @@ type Props = {
   addSuffix?: boolean;
   shorten?: boolean;
   relative?: boolean;
-  format?: string;
+  format?: object;
```

Comment:
This can be typed so that the object keys have to be valid locale strings. Something similar to:

```suggestion
  format?: Record<keyof locales, string>;
```

Reply:
That one is giving me a hard time. I don't understand where I can find the locales object definition to get its keys with keyof ? I made several attemps (see last ocmmits below) without any success for now